internal/fabric/tx.go

```diff
@@ -74,20 +75,7 @@ func buildTX(signer, channelID, chaincodeName, function string, args []string) (
 
 // GetTXReceipt gets the receipt for the transaction
 func (tx *Tx) GetTXReceipt(ctx context.Context, rpc RPCClient) (bool, error) {
-	start := time.Now().UTC()
-
-	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
-	defer cancel()
-
-	result, err := rpc.Invoke(ctx, tx.ChannelID, "qscc", "GetTransactionByID", []string{tx.Hash})
-	if err != nil {
-		return false, errors.Errorf(errors.RPCCallReturnedError, "GetTransactionByID", err)
-	}
-	tx.Receipt = result
-	callTime := time.Now().UTC().Sub(start)
 	isMined := tx.Receipt.BlockNumber > 0
-	log.Debugf("GetTransactionByID(%x,latest)=%t [%.2fs]", tx.Hash, isMined, callTime.Seconds())
-
```

Comment:
Is this really all that is required to get the receipt now?

Reply:
yep, since the receipts for the Fabric transactions are obtained via the event, there is no need to poll the node for the receipt